README.md

```diff
@@ -123,6 +127,24 @@ Use this header in an Objective-C project by adding
 
 to your implementation or header files. Note that spaces in your project name result in underscores. So `Your Project` becomes `Your_Project-Swift.h`.
 
+## Branches
```

Comment:
A list would be sufficient or we add more description. Right now it's too much heading for not enough text in my opinion.